lib/susedistribution.pm

```diff
@@ -357,15 +362,17 @@ sub activate_console {
         }
     }
 
-    if ($console =~ m/^(.*)-console/) {
-        my ($name, $user) = ($1, $1);
-        if ($name eq 'user') {
-            $user = $testapi::username;
-        }
-        elsif ($name eq 'log') {
-            $user = 'root';
-        }
+    # TODO: We should not be extracting this information from the console name,
```

Comment:
In general I think that information should be kept in the language's data structures where possible. In this case I think there should be a hash map associated with each console name containing attributes and callbacks for each console.

I will just remove the TODO though, it's not important for the LTP.